test/python/transpiler/test_mappers.py

```diff
@@ -106,12 +106,6 @@ def create_passmanager(self, coupling_map, initial_layout=None):
             passmanager.append(SetLayout(Layout(initial_layout)))
 
         with warnings.catch_warnings():
-            # TODO: remove this filter when StochasticSwap is removed
-            warnings.filterwarnings(
-                "ignore",
-                category=DeprecationWarning,
-                message=r".*StochasticSwap.*",
-            )
             # pylint: disable=not-callable
             passmanager.append(self.pass_class(CouplingMap(coupling_map), **self.additional_args))
```

Comment:
Can remove the whole `warnings.catch_warnings` now too

Reply:
Fixed in https://github.com/Qiskit/qiskit/pull/13791/commits/1063f06284bdbb7fc41a1b1f3272cef7d0c5c1d5